packages/core/lib/engine_http.js

```diff
@@ -350,8 +350,31 @@ HttpEngine.prototype.step = function step(requestSpec, ee, opts) {
 
         // Request.js -> Got.js translation
         if (params.qs) {
-          requestParams.searchParams = template(params.qs, context);
+          const searchParams = template(params.qs, context);
```

Comment:
Question: any reason not to use `querystring` here like this:

```js
const searchParams = querystring.stringify(template(params.qs, context));
```

Reply:
I was trying to avoid using `querystring` due to its legacy status. 
Unfortunately the `URLSearchParams` doesn't really have an equivalent to the `querystring.stringify`. I hadn't noticed it had an `.append()` option before, so I can improve the code a bit now:

```js
if (params.qs) {
  const searchParams = template(params.qs, context);
  requestParams.searchParams = new URLSearchParams();

  _.map(searchParams, (value, key) => {
    if (Array.isArray(searchParams[key])) {
      _.map(value, (item) => requestParams.searchParams.append(key, item));
    } else {
      requestParams.searchParams.append(key, value);
    }
  });
}
```


Though of course `querystring.stringify()` would be a one liner (I tested it), but it is technically legacy... 

Let me know what you'd prefer I use @hassy